templates/topic/reports/a11y-1.json

```diff
@@ -0,0 +1,689 @@
+      },
+      "earl:result": [
+        {
+          "earl:test": "act:rules/wcag2x/sc/1_1_1_situation_a",
+          "earl:outcome": "earl:passed",
+          "earl:mode": "earl:manual",
+          "earl:subject": "_non-text-content_itm1",
+          "dct:description": "Image includes an alt tag to describe the image content."
+        }
+      ],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed",
+      "selected": "act:rulesets/wcag2x/sc/non-text-content#situations"
+    },
+    {
+      "earl:test": "WCAG21:audio-only-and-video-only-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": "act:rulesets/wcag2x/sc/non-text-content#situations"
+    },
+    {
+      "earl:test": "WCAG21:captions-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:audio-description-or-media-alternative-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:captions-live",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:audio-description-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:info-and-relationships",
+      "earl:subject": {
+        "@id": "_info-and-relationships_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": ["earl:TestSubject", "schema:WebPageElement"],
+        "dct:description": "The page provides semantic structure, including headings and lists, to make information and relationships clear and accessible."
```

Comment:
This description are the refined of the subject, not the description of this test. 

The description of this test are at line 146

Reply:
Comments addressed